### PR TITLE
Update README for release

### PR DESCRIPTION
**Pull request by tschaffter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,52 +8,60 @@
...
 An example implementation of the [NLP Sandbox Date Annotator].
 
-## Specification
+## Overview
 
-- Implements the [NLP Sandbox Date Annotator OpenAPI specification]
-- Annotates date strings in clinical notes using simple regular expressions
+This repository provides a Python-Flask implementation of the [NLP Sandbox Date
+Annotator]. The Date Annotator is one of the first NLP Tools that can be
+benchmarked on nlpsandbox.io. A Date Annotator takes as input one clinical notes
```
Comment (vpchung) [inline README.md:15]:
```suggestion
benchmarked on nlpsandbox.io. A Date Annotator takes as input one clinical note,
```

File: README.md [hunk 2]
```diff
@@ -8,52 +8,60 @@
...
+as well as information about the patient and outputs a list of predicted date
```
Comment (vpchung) [inline README.md:16]:
```suggestion
as well as information about the patient, and outputs a list of predicted date
```

File: README.md [hunk 3]
```diff
@@ -8,52 +8,60 @@
...
+annotations found in the clinical note.
 
-NLP Sandbox API implemented:
+- Date Annotator API version: 0.3.1
```
Comment (vpchung) [inline README.md:19]: Maybe include a header for these specs? e.g.

---

#### Specification
- Date Annotator API version: 0.3.1
- Tool version: 0.3.2
- Docker image: [nlpsandbox/date-annotator-example]

File: README.md [hunk 4]
```diff
@@ -8,52 +8,60 @@
...
-## Annotating clinical notes
-
-When running, the Date Annotator provides a web interface (http://localhost:8080/api/v1/ui/)
-that you can use to explore the input, output and actions available.
+### Annotating clinical notes
 
-## Evaluating performance in the NLP Sandbox
+The Date Annotator provides a web interface that you can use to annotate
+clinical notes. The address of this interface depends on whether you run the
+Date Annotator using Docker (production mode) or using Python development
```
Comment (vpchung) [inline README.md:60]:
```suggestion
Date Annotator using Docker (production mode) or the Python development
```

File: README.md [hunk 5]
```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
 - [Node JS](https://nodejs.org/)
 - Java (required by [OpenAPITools/openapi-generator])
 
-### Start developing your own server
+### Creating a new GitHub repository
+
+Depending on the programming language-framework
```
Comment (vpchung) [inline README.md:78]: Maybe:

"This step will depend on your preferred programming language-framework."

File: README.md [hunk 6]
```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
...
+
+- If you develop in Python-Flask, create a new repository from this [GitHub
+  template].
+- If you develop in Java-Spring, create a new repository from the GitHub
+  template [nlpsandbox/date-annotator-example-java].
+
+If you prefer to developed using another language or if you want to learn how
```
Comment (vpchung) [inline README.md:85]:
```suggestion
If you prefer to develop using another language or if you want to learn how
```

File: README.md [hunk 7]
```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
...
-Start by downloading the latest version of the [NLP Sandbox Date Annotator OpenAPI specification]
-and save this file to the root folder of your project.
+The CI/CD workflow builds and pushes the following tags to the docker registry:
 
-    curl -f -O https://sage-bionetworks.github.io/nlp-sandbox-schemas/date-annotator/latest/openapi.yaml
+- The tag `edge` is created when a commit is pushed to the default branch of
+  this repository (`develop`).
+- The tags `latest`, `x`, `x.y`, `x.y.z` are created when the GitHub release
+  `x.y.z` is created.
+- The tag `nighlty` is created every night everyday at 10am UTC.
```
Comment (vpchung) [inline README.md:123]: Typo? `nighlty` -> `nightly`

File: README.md [hunk 8]
```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
...
 
-Generate the initial server stub from the OpenAPI specification
+- `Date Annotator API version`: The version of the Date Annotator specification
+  implemented by the tool.
+- `Tool version`: The version of the tool.
 
-    mkdir server
-    ./node_modules/.bin/openapi-generator-cli \
-        generate -i openapi.yaml -g spring -o server
+Initially, it may be tempating to align the tool version to the API version. As
```
Comment (vpchung) [inline README.md:220]:
```suggestion
Initially, it may be tempting to align the tool version to the API version. As
```

File: README.md [hunk 9]
```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
...
+you improve your tool and fixes bugs, you will likely release more than one
```
Comment (vpchung) [inline README.md:221]:
```suggestion
you improve your tool and fix bugs, you will likely release more than one
```

File: README.md [hunk 10]
```diff
@@ -65,126 +73,174 @@ you can then submit for evaluation to the NLP Sandbox.
...
+Visit nlpsandbox.io for instructions on how to submit your tool and evaluate its
+performance on public and private datasets.
 
-The API documentation UI is now available at http://localhost:8080.
+## License
 
+[Apache License 2.0]
 
-<!-- Definitions -->
+<!-- Links -->
```
Comment (vpchung) [inline README.md:234]: Not sure if this is on purpose but this section is still hidden from view.

File: README.md [hunk 11]
```diff
@@ -8,52 +8,60 @@
...
-| [Date Annotator](/openapi/date-annotator) | `nlpsandbox:date-annotator` | 0.3.2 |
+## Model
 
-## Usage
+This NLP tool relies on simple regular expressions to identify the location of
+date strings in a clinical note.
+
+This implementation is provided as an example that Developers can use to quick
+start the development of a new model by creating a repository from this [GitHub
+template] (see below). The Docker image built and published automatically by the
```
Comment (vpchung) [inline README.md:30]: This sentence is a bit run-on. Maybe something like this instead?

"The CI/CD workflow of this repository will automatically build and publish a Docker image to DockerHub.  The model can then be submitted as-is, if you wish to benchmark its performance -- just don't expect a high performance!"

File: README.md [hunk 12]
```diff
@@ -8,52 +8,60 @@
...
 
-[![Gitpod](https://img.shields.io/badge/Gitpod-Open_in_Gitpod-blue?color=0273b3&labelColor=555555&logoColor=ffffff&style=for-the-badge&logo=gitpod&label=)](https://gitpod.io/#https://github.com/nlpsandbox/date-annotator-example)
+## Usage
 
 ### Running with Docker
 
 The command below starts the Date Annotator locally.
 
-    docker-compose up
+    docker-compose up --build
```
Comment (vpchung) [inline README.md:41]: Maybe add a comment on how to stop the run?

"You can stop the container run with `^C`, followed by `docker-compose down`."

File: README.md [hunk 13]
```diff
@@ -8,52 +8,60 @@
 
 An example implementation of the [NLP Sandbox Date Annotator].
```
Comment (vpchung) [inline README.md:9]: This link is broken for me?

File: README.md [hunk 14]
```diff
@@ -8,52 +8,60 @@
...
 
-## Specification
+## Overview
 
-- Implements the [NLP Sandbox Date Annotator OpenAPI specification]
-- Annotates date strings in clinical notes using simple regular expressions
+This repository provides a Python-Flask implementation of the [NLP Sandbox Date
+Annotator]. The Date Annotator is one of the first NLP Tools that can be
```
Comment (vpchung) [inline README.md:14]: Broken link

File: README.md [hunk 15]
```diff
@@ -8,52 +8,60 @@
...
 
-## Usage
+This NLP tool relies on simple regular expressions to identify the location of
+date strings in a clinical note.
+
+This implementation is provided as an example that Developers can use to quick
+start the development of a new model by creating a repository from this [GitHub
+template] (see below). The Docker image built and published automatically by the
+CI/CD workflow of this repository can be submitted "as it is" to the [NLP
+Sandbox] to benchmark its performance if you wish to try submitting first (don't
```
Comment (vpchung) [inline README.md:32]: Broken link

**Review by vpchung — CHANGES_REQUESTED** [review]
Overall, looks good! Just some minor fixes and a couple sentence modifications.

There are also some broken links but I'm not sure if maybe they'll be fixed after the CI/CD workflow runs?

File: README.md (hunk 3 above)
Comment (tschaffter) [reply README.md:19]: I added the header

File: README.md (hunk 5 above)
Comment (tschaffter) [reply README.md:78]: Done

File: README.md (hunk 7 above)
Comment (tschaffter) [reply README.md:123]: Fixed

File: README.md (hunk 10 above)
Comment (tschaffter) [reply README.md:234]: Yes. Everything below this section is not visible.

File: README.md (hunk 11 above)
Comment (tschaffter) [reply README.md:30]: Done

File: README.md (hunk 12 above)
Comment (tschaffter) [reply README.md:41]: Added the sentence (I used `Ctrl+C`)

File: README.md (hunk 14 above)
Comment (tschaffter) [reply README.md:14]: Fixed

File: README.md (hunk 13 above)
Comment (tschaffter) [reply README.md:9]: Fixed

File: README.md (hunk 15 above)
Comment (tschaffter) [reply README.md:32]: Fixed

**Comment by tschaffter** [comment]
@vpchung Thanks for the review!

**Pull request merged**